real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -265,8 +287,13 @@ int main(void) {
     // 16k ohm on top.
     // Calculated map normally set to 13000, but the avcc is 4.86 volts
     // rather than 5.
-    g_current_voltage = map_signal(adc_read_blocking(0), 0, 255, 0, 12636); // in millivolts
+    g_current_voltage = adc_read_blocking(BATTERY_ADC);
+    g_current_voltage = map_signal(g_current_voltage, 0, 255, 0, 12636); // in millivolts
     
+    // Read/convert steering pot
+    g_steering_feedback = adc_read_blocking(STEERING_POT_ADC);
+    g_steering_feedback = map_signal(g_steering_feedback, 0, 255, -10000, 10000);
```

Comment:
Yeah. This part of the pull request needs to be... calibrated. I'll convert to defines.